serverless.yml

```diff
@@ -0,0 +1,65 @@
+service: superagent-aws-signed-request-test
+
+plugins:
+   - serverless-plugin-include-dependencies
+   - serverless-resources-env
+
+provider:
+  name: aws
+  runtime: nodejs4.3
+  cfLogs: true
+
+package:
+  exclude:
+    - src/**
+    - tests/**
+    - node_modules/**
+  include:
+    - tests/infra/**
+
+custom:
+  resource-output-dir: /tests/infra/resources
```

Comment:
This was the format the plugin used, and it works, but I can try without leading slash